doc/man1/openssl-cmp.pod.in

```diff
@@ -990,16 +990,18 @@ to issue the following shell commands.
 
   cd /path/to/openssl
   export OPENSSL_CONF=openssl.cnf
+=begin comment
   wget 'http://pki.certificate.fi:8080/install-ca-cert.html/ca-certificate.crt\
         ?ca-id=632&download-certificate=1' -O insta.ca.crt
-  openssl genrsa -out insta.priv.pem
-  openssl cmp -section insta
+=end comment
+  ./openssl genrsa -out insta.priv.pem
+  ./openssl cmp -section insta
 
 This should produce the file F<insta.cert.pem> containing a new certificate
 for the private key held in F<insta.priv.pem>.
 It can be viewed using, e.g.,
 
-  openssl x509 -noout -text -in insta.cert.pem
+  ./openssl x509 -noout -text -in insta.cert.pem
```

Comment:
Why `./` here and everywhere?

I expect that most people use the system's openssl which is in their path.

Reply:
I added the `./` to make sure that the `openssl` binary used is the one consistent with the `./openssl.cnf` and `./insta.ca.crt` files - note the `cd /path/to/openssl` at the beginning of the example.
Yet I agree that for most users (the right version of) the binary should be in the path, so I've dropped those `./` again.